terraform/sandbox/main.tf

```diff
@@ -31,6 +31,20 @@ module "redis" {
   redis_plan_name  = "redis-dev"
 }
 
+module "redis-v70" {
```

Comment:
do we want 'v70' or do we want 'new'?

Reply:
I was planning to keep `redis-v70` around after the resource `redis` is deleted. This is just the "[local name](https://developer.hashicorp.com/terraform/language/modules/syntax#calling-a-child-module)" of the resource, which is arbitrary. Using `v70` means next we can use `v71` next time, instead of `new-new`.

Reply:
Great question, @terrazoon!  @jskinne3 has the right of it here; by including the version in the module name, which only Terraform cares about in this context, we can add future versions when they become available to go through the same process to upgrade again.

Also, as nice as it would be to rename the existing module to something like `redis-old` to indicate it should be flagged for deletion later on, and name the new module as just `redis`, I strongly suspect Terraform would see that as a change to existing infrastructure and do things we do not want it to do. 😬